README

```diff
@@ -3,6 +3,7 @@ README
 
 Compiling:
     automake
+    autoreconf -i

```

Comment:
`autoreconf` will launch `automake` and `autoconf` so they are redundant.  Better only keep `autoreconf`, followed by `./configure` and `make` :wink: